tests/functional/test_install_reqs.py

```diff
@@ -106,6 +106,33 @@ def test_multiple_requirements_files(script, tmpdir):
     assert script.venv / 'src' / 'initools' in result.files_created
 
 
+@pytest.mark.network
+def test_remote_requirements_file_stuff(script):
+    """
+    Test installing from a simple remote requirements file
+    """
+    # this requirements file just contains a comment
+    url = ('https://raw.githubusercontent.com/pypa/pip/6.0.8/'
+           'dev-requirements.txt')
```

Comment:
Unsure, is it testing something that isn't otherwise tested? If it is then no, if it's not then we can get rid of it.